src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -66,6 +72,9 @@ protected QueryOuterClass.Query getCreateTable(String keyspace, String table) {
               + vectorSize
               + ">, "
               + "    PRIMARY KEY (key))";
+      if (vectorize != null) {
+        createTableWithVector = createTableWithVector + " WITH comment = '" + vectorize + "'";
```

Comment:
Not sure if it matters, but if `vectorize` is serialized JSON it could still have characters that require escaping (esp. apostrophes themselves). Although I guess those would lead to CQL query failure, not corrupted data.

Reply:
Not sure what kind of characters to handle since this is just a text value, If needed I will add permitted character for the vectorize options.